ITEM.tsv

```diff
@@ -5992,11 +5992,11 @@ ITEM_20150804_005991	Title - Ausrine's Incarnation
 ITEM_20150804_005992	A title given to the first player who has reached the 6th rank during the 3rd CBT. Right-click to use.
 ITEM_20150804_005993	Title - Ausrine's Hero
 ITEM_20150804_005994	A title given to players between 2~10 who have reached the 6th rank during the 3rd CBT.
-ITEM_20150804_005995	The 1st Symbol of Trial of the Great King
+ITEM_20150804_005995	1st Symbol of Ordeal of the Great King
```

Comment:
An ordeal is a difficult test. While everyone can interpret what the player is doing is either hard or easy, it remains a fact that Zachariel prepared four difficult tests that only the Revelator could pass.